README.md

```diff
@@ -86,6 +94,12 @@ dependencyTrackCompanion {
         projectName.set(name)
         projectVersion.set(version)
     }
+    riskScore{
+        projectName.set(name)
+        projectVersion.set(version)
+        timeout.set(20)
```

Comment:
Please use kotlin duration. Without any unit of measurement it's unclear if we're talking about seconds, milliseconds or something else ;)